setup/src/Magento/Setup/Test/Unit/Module/Di/Code/Reader/ClassesScannerTest.php

```diff
@@ -22,6 +22,6 @@ public function testGetList()
         $pathToScan = str_replace('\\', '/', realpath(__DIR__ . '/../../') . '/_files/app/code/Magento/SomeModule');
         $actual = $this->model->getList($pathToScan);
         $this->assertTrue(is_array($actual));
-        $this->assertCount(5, $actual);
+        $this->assertCount(6, $actual);
```

Comment:
I ran into this same issue (also because of the ::class thing) on #8965.  If it *is* the same thing then this test should remain with a count of 5.